finetrainers/dataset.py

```diff
@@ -237,7 +246,7 @@ def _preprocess_video(self, path: Path) -> Tuple[torch.Tensor, Optional[torch.Te
         video_reader = decord.VideoReader(uri=path.as_posix())
         video_num_frames = len(video_reader)
 
-        indices = list(range(0, video_num_frames, video_num_frames // self.max_num_frames))
+        indices = list(range(0, video_num_frames))
```

Comment:
I expected that in ImageOrVideoDataset, since it's not resizing, it shouldn't limit the number of frames, either? But in hindsight, maybe it should, since max_num_frames is set.